test/start-stop.make

```diff
@@ -1,68 +1,27 @@
+		KUBECTL="$(PWD)/_work/$(CLUSTER)/ssh-$(CLUSTER) kubectl" PATH='$(PWD)/_work/bin/:$(PATH)' ./test/setup-ca-kubernetes.sh && \
 		touch _work/$(CLUSTER)/secretsdone; \
-	fi
-	_work/$(CLUSTER)/ssh kubectl version --short | grep 'Server Version' | sed -e 's/.*: v\([0-9]*\)\.\([0-9]*\)\..*/\1.\2/' >_work/$(CLUSTER)/kubernetes.version
-	( . test/test-config.sh && _work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-csi-$${TEST_DEVICEMODE}-testing.yaml )
-	_work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-storageclass-ext4.yaml
-	_work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-storageclass-xfs.yaml
-	_work/$(CLUSTER)/ssh kubectl apply -f - <deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-storageclass-cache.yaml
-	@ echo
-	@ echo "The test cluster is ready. Log in with _work/$(CLUSTER)/ssh, run kubectl once logged in."
+	fi \
+	&& test/setup-deployment.sh
+	@ echo "The test cluster is ready. Log in with _work/ssh-$(CLUSTER), run kubectl once logged in."
 	@ echo "Alternatively, KUBECONFIG=$$(pwd)/_work/$(CLUSTER)/kube.config can also be used directly."
 	@ echo "To try out the pmem-csi driver persistent volumes:"
-	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-pvc.yaml | _work/$(CLUSTER)/ssh kubectl create -f -"
-	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-app.yaml | _work/$(CLUSTER)/ssh kubectl create -f -"
+	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-pvc.yaml | _work/$(CLUSTER)/ssh-$(CLUSTER) kubectl create -f -"
+	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-app.yaml | _work/$(CLUSTER)/ssh-$(CLUSTER) kubectl create -f -"
 	@ echo "To try out the pmem-csi driver cache volumes:"
-	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-pvc-cache.yaml | _work/$(CLUSTER)/ssh kubectl create -f -"
-	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-app-cache.yaml | _work/$(CLUSTER)/ssh kubectl create -f -"
+	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-pvc-cache.yaml | _work/$(CLUSTER)/ssh-$(CLUSTER) kubectl create -f -"
+	@ echo "   cat deploy/kubernetes-$$(cat _work/$(CLUSTER)/kubernetes.version)/pmem-app-cache.yaml | _work/$(CLUSTER)/ssh-$(CLUSTER) kubectl create -f -"
 
 stop:
-	for i in $$(seq 0 $$(($(NUM_NODES) - 1))); do \
-		if [ -e _work/$(CLUSTER)/qemu.$$i.pid ]; then \
-			kill -9 $$(cat _work/$(CLUSTER)/qemu.$$i.pid) 2>/dev/null; \
-			rm -f _work/$(CLUSTER)/qemu.$$i.pid; \
-		fi; \
-	done
+	. test/test-config.sh && \
+	_work/$(CLUSTER)/stop.sh
+
+restart:
+	. test/test-config.sh && \
+	_work/$(CLUSTER)/restart.sh
```

Comment:
Same here: if `	_work/$(CLUSTER)/restart.sh` does not exist, print an explicit error message.